cpu/samd5x/periph/can.c

```diff
@@ -824,115 +828,155 @@ static int _set(candev_t *candev, canopt_t opt, void *value, size_t value_len)
     return res;
 }
 
+void _mcan_hdr_can_frame(struct can_frame * frame, uint32_t fe_r0, uint32_t fe_r1){
```

Comment:
```suggestion
static void _mcan_hdr_can_frame(struct can_frame * frame, uint32_t fe_r0, uint32_t fe_r1)
{
```

IMO it is not obvious by the name what this function does. Do you mind to add a 1-2 line comment to explain it?